[2023-03-01 13:10:05,772] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-08T16:05:00+00:00 [queued]>
[2023-03-01 13:10:05,855] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-08T16:05:00+00:00 [queued]>
[2023-03-01 13:10:05,857] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:10:05,858] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 13:10:05,858] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:10:05,994] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-12-08T16:05:00+00:00
[2023-03-01 13:10:06,048] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-12-08T16:05:00+00:00', '--job-id', '4911', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmpbrypv7x5', '--error-file', '/tmp/tmp2p6ruyjo']
[2023-03-01 13:10:06,055] {standard_task_runner.py:77} INFO - Job 4911: Subtask get_name
[2023-03-01 13:10:06,018] {standard_task_runner.py:52} INFO - Started process 20483 to run task
[2023-03-01 13:10:06,351] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-12-08T16:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 13:10:06,650] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-12-08T16:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-08T16:05:00+00:00
[2023-03-01 13:10:06,805] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 13:10:06,887] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221208T160500, start_date=20230301T131005, end_date=20230301T131006
[2023-03-01 13:10:07,083] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-03-01 13:10:07,132] {local_task_job.py:146} INFO - Task exited with return code 0
